scale-ui/app/modules/jobs/controllers/jobTypesErrorRatesController.js

```diff
@@ -64,6 +88,20 @@
                 }
             });
         };
+        
+        vm.setStyle = function (colField, errorType) {
+            var errorValue = (colField[errorType] / colField.total).toFixed(2);
+            var textColor = errorValue >= 0.5 ? '#fff' : '#000';
+            var rgb = errorType === 'system' ? '103, 0, 13' : errorType === 'algorithm' ? '203, 24, 29' : '241, 105, 19';
```

Comment:
should this be set in css and use classes instead of hard coding the color values?